.github/workflows/upload.yml

```diff
@@ -0,0 +1,25 @@
+name: Linux_Node_Workflow_ScmCred
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: 'run npm'
+      run: |
+        npm install
+        npm run build --if-present
+        npm run test --if-present
+    - uses: Azure/functions-action@v1
+      id: fa
+      with:
+        app-name: TypeScriptReposAutomation
+        publish-profile: ${{ secrets.SCM_CREDENTIALS }}
```

Comment:
This _should_ automatically deploy the app on every push to master, I've set up all the credentials etc